addons/website/static/src/snippets/s_image_gallery/000.js

```diff
@@ -86,7 +86,14 @@ const GallerySliderWidget = publicWidget.Widget.extend({
         this.$prev = this.$indicator.find('li.o_indicators_left').css('visibility', ''); // force visibility as some databases have it hidden
         this.$next = this.$indicator.find('li.o_indicators_right').css('visibility', '');
         var $lis = this.$indicator.find('li[data-slide-to]');
-        var nbPerPage = Math.floor(this.$indicator.width() / $lis.first().outerWidth(true)) - 3; // - navigator - 1 to leave some space
+        var indicator_width = this.$indicator.width();
+        if (!indicator_width){
+            var clone = this.$indicator.clone();
+            $('body').append(clone);
```

Comment:
`body` is not good as the page layout could be boxed. Instead of clone and adding elsewhere, you should just be able to find the invisible parent and force it to be visible. That would not solve all cases but that would be good enough.